docker-compose.yml

```
@@ -1,29 +1,38 @@
+    ports:
+      - "9501:9501"
+      - "9503:9503"
+    environment:
+      - TZ=Asia/Shanghai
+      - APP_NAME=MineAdmin
+    command:
+      - sh
+      - -c
+      - |
+        ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
+        tail -F /dev/null
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)